### DISCUSSION TITLE
https://api.allorigins.win/ down?

### DISCUSSION THREAD

**Discussion by michaelkowalczyk** [LiaScript/LiaScript#211] [Q&A]
Hi,
it seems that today our Liascript courses do not load anymore when accessing via hyperlink. The .md hosting server is available. Is there a chance that https://api.allorigins.win/ is down at some point? Is there a way to circumvent this and have access withhout the allorigins intermediary?
Thank you and best regads
Michael

**Comment by andre-dietrich** [comment]
I think this might happen from time to time, this is a proxy and a simple way to circumvent CORS errors ... where are your courses hosted?

**Reply by michaelkowalczyk** [reply]
Yes, that is kind of a bummer. 
Our .md and image files are on a technically limited lima-city.de webserver. I see some use "https://raw.githubusercontent.com" as intermediary. Would that be a less problematic one?

**Reply by michaelkowalczyk** [reply]
Would you have time for quick call in Germany? If so please mail me michael.kowalczyk@uni.kn. Thanks :)

**Reply by andre-dietrich** [reply]
Hosting on GitHub works perfect, this can also be used for images, videos and even javascript and css ... this will not require a proxy in between ...
